backend/app/api/error_handlers.py

```
@@ -0,0 +1,71 @@
+"""
+에러 핸들러 (UTI-005)
+"""
+from fastapi import Request, status
+from fastapi.responses import JSONResponse
+from fastapi.exceptions import RequestValidationError
+from starlette.exceptions import HTTPException as StarletteHTTPException
+
+
+async def not_found_handler(request: Request, exc: StarletteHTTPException):
```

Ruff (0.14.5)

Unused function argument: `request`

(ARG001)